app/assets/javascripts/polaris_view_components/autocomplete_controller.js

```diff
@@ -4,43 +4,49 @@ import { debounce } from './utils'
 
 export default class extends Controller {
   static targets = ['popover', 'input', 'results', 'option', 'emptyState']
-  static values = { url: String, selected: Array }
+  static values = { multiple: Boolean, url: String, selected: Array }
 
-  connect() {
-    this.inputTarget.addEventListener("input", this.onInputChange)
-  }
+  // Actions
 
-  disconnect() {
-    this.inputTarget.removeEventListener("input", this.onInputChange)
+  onFocus() {
+    this.showAutocomplete()
   }
 
-  // Actions
-
-  toggle() {
```

Comment:
I renamed this to `showAutocomplete` as this wasn't really "toggling", but only displaying/showing the autocomplete suggestions